pandas/tests/frame/test_analytics.py

```diff
@@ -595,7 +606,8 @@ def test_describe_empty_categorical_column(self):
         df = pd.DataFrame({"empty_col": Categorical([])})
         result = df.describe()
         expected = DataFrame({'empty_col': [0, 0, None, None]},
-                             index=['count', 'unique', 'top', 'freq'])
+                             index=['count', 'unique', 'top', 'freq'],
+                             dtype='object')
```

Comment:
This was added in 0.25.0. Returning object dtype isn't an (additional) API change from 0.24

Reply:
should these be np.nan? (and not None); I am not sure we actually check this in the comparator.

Reply:
It wasn't clear to me whether I should use None or NaN. Arguments for NaN instead of None

1. it's our most consistent missing value marker
2. We often translate None to NaN implicitly

but I don't know what's best.

Reply:
right I think we should np.nan; however I don't remember if we actually test this in assert_series_equal; maybe the check_exact arg does this but I don't remember.

Reply:
nice catch

```python
In [4]: a = pd.Series([None], dtype=object)

In [5]: b = pd.Series([np.nan], dtype=object)

In [6]: pd.util.testing.assert_series_equal(a, b)

```